reducers/gotPlayerMatches.js

```diff
@@ -0,0 +1,30 @@
+import { playerMatchesActions } from '../actions';
```

Comment:
Is it possible to combine all the gotPlayer* reducers into a single one that takes an additional param `tab` and extracts the necessary properties based on that?